Source/JavaScriptCore/inspector/scripts/codegen/generator.py

```diff
@@ -52,6 +52,7 @@ def ucfirst(str):
     'canvas-webgl2': 'CanvasWebGL2',  # Recording.Type.canvas-webgl2
     'webgl': 'WebGL',  # Canvas.ContextType.webgl
     'webgl2': 'WebGL2',  # Canvas.ContextType.webgl2
+    'webgpu': 'WebGPU',  # Canvas.ContextType.webgpu
```

Comment:
Fly by? Seems fine, but weird it was missing?

Reply:
yeah.  i think this was (accidentally?) removed when the first WebGPU implementation was ripped out.  without this, regenerating the old files results in `"Webgpu"` instead of `"WebGPU"`